ega-charts/localega/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.rbacEnabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: {{ template "localega.fullname" . }}-serviceAccount
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: lega
```

Comment:
side note, would be nice to make this custom, but not required as of now.

Reply:
True, we should be able to use the release name.